pages/index.js

```diff
@@ -37,6 +37,7 @@ export default function IndexPage(props) {
   const [workflow, setWorkflow] = useState();
   const [_loading, setLoading] = useState(false);
   const [actionError, _setActionError] = useState(null);
+  const [hideAlerts, setHideAlerts] = useState(true);
```

Comment:
how about renaming this to `firstLoad` or `initLoad`. Then later in the logic it would be more meaningful:
```
!loading && workflow.message && isLatestTask && !initLoad &&
```

Reply:
Yes! I tried a few different variable names but didn't think of `init`
I've done this now.